pom.xml

```diff
@@ -119,7 +119,7 @@
     <mesos.classifier>shaded-protobuf</mesos.classifier>
     <slf4j.version>1.7.5</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
-    <hadoop.version>1.0.4</hadoop.version>
+    <hadoop.version>2.4.0</hadoop.version>
```

Comment:
I'd guess that any `hadoop.version >= 2` would work for my/intellij's purposes, but I'll double-check that